README.md

```diff
@@ -1,42 +1,124 @@
-# GenSX Monorepo
+# GenSX ⚡️
 
-This monorepo contains GenSX and its related packages - a framework for building LLM workflows and AI agents with JSX on the backend. Every GenSX component is a pure function, making it easily shareable by default.
+[![npm version](https://badge.fury.io/js/gensx.svg)](https://badge.fury.io/js/gensx)
+[![Website](https://img.shields.io/badge/Visit-gensx.dev-orange)](https://gensx.dev)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## Repository Structure
+[GenSX](https://gensx.dev/overview/) is a simple typescript framework for building complex LLM applications. It’s built around functional, reusable components that are composed using JSX to create and orchestrate workflows.
```

Comment:
Wondering if we should leave out JSX from intro completely. 

If we do mention it, we should probably adjust the final few words to be "and orchestrate backend workflows" to make sure that no one is confused and bounces thinking this is a client side framework.

Reply:
easy enough to drop the `using JXS` bit; i'll remove it